recipes/python/backup-restore/README.md

```diff
@@ -0,0 +1,130 @@
+# NetBackup VmWare agentless single and group VM backup and restore APIs code samples
+
+## Executing the scripts:
+
+Pre-requisites:
+- NetBackup 8.3 or higher
+- Python 3.6 or higher
+- Python modules: `requests`
+
+Use the following commands to run the scripts.
+### - Single VM backup and restore
+
+This single_vm_backup_restore.py script demonstrates how to backup a VM (VMware virtual machine) using a protection plan and restore it using NetBackup APIs.
```

Comment:
Can you also mention that the restores are instant access restores?

Reply:
Done.